src/service/BookingService.java

```diff
@@ -30,6 +31,25 @@ public BookingService(BookingDataDAO bookingDataDAO,GuestDataDAO guestDataDAO) {
 	}
 
 	public Integer saveBooking(LocalDateTime entryDate, LocalDateTime departureDate, PaymentMethodDTO methodPayment)  {
+		if (entryDate == null) {
+			throw new IllegalArgumentException("Entry date was null");
+		}
+		if (departureDate == null) {
+			throw new IllegalArgumentException("Departure date was null");
+		}
+		if (methodPayment == null) {
+			throw new IllegalArgumentException("Method payment was null");
+		}		
+		if (departureDate.toLocalDate().isBefore(entryDate.toLocalDate())){
+			throw new IllegalArgumentException("Entry date can't be before departure date");
+		}
+		if (departureDate.toLocalDate().isEqual(entryDate.toLocalDate())){
```

Comment:
Please create a test in which `departureDate="20230828-12:00:01"` and `entryDate=20230828-12:00:00`, I need the system throws the exception `throw new IllegalArgumentException("Entry date can't be equal to departure date");`